doc/HOWTO_forks.md

```diff
@@ -41,3 +41,19 @@ the commits in your branch so you only need to bring one commit.
 git fetch nu
 git cherry-pick XXXXX
 ```
+
+# Deploy
+From the root of your fork:
+
+- `bundle exec cap [staging_name] deploy`
+
+Find `staging_name` by the name of the `.rb` files in `config/deploy`
```

Comment:
It's a little funky. For NU we using `staging` and `prod`, UIC, we have `uic_stage`/`uic_prod` (I forget exactly why), then DC we have `staging` and `txi_staging` (because we have the stage that's auto-deployed in our environment, and then the one in theirs). It is unrelated to `rails_env`.